reference/toaztr.v1.yaml

```diff
@@ -739,6 +739,38 @@ paths:
       security:
         - toaztr_auth:
             - cases/read
+  /v1/cases/by-status:
+    get:
+      operationId: getCasesByStatus
+      tags:
+        - case
+      description: 'Liste les dossiers par leur status, la recherche n''est pas sensible à la casse'
+      parameters:
+        - $ref: '#/components/parameters/page[size]'
+        - $ref: '#/components/parameters/page[cursor]'
+        - $ref: '#/components/parameters/ordering'
+        - schema:
+            type: array
```

Comment:
Isn't it missing the type within the array?
I understand why you would want to filter by a list of statuses, but I don't think there is any index structure that can make that happen "efficiently" within DynamoDB.

Reply:
The idea is to filter all the status in theses groups in only 3 calls:
```
  newStatuses = [CaseStatus.New, CaseStatus.Instruction];
  ongoingStatuses = [CaseStatus.SentToBank, CaseStatus.GrantedBank, CaseStatus.CustomerAccepted, CaseStatus.Completed];
  doneStatuses = [CaseStatus.Closed, CaseStatus.Invoiced, CaseStatus.Canceled];
```

If not efficient, I'll just do several call filtering by 1 status each, no problem.

Reply:
For the type I don't have the possibility to fill it in stoplight :/
![image](https://user-images.githubusercontent.com/430679/142282760-9b5155b2-5e78-43f7-bf90-9b41afbe427d.png)